src/providers/solr-search.php

```diff
@@ -70,27 +69,25 @@ function parse_query_response( $responseString ) {
     $docs = $parsedResult->response->docs;
     foreach ( $docs as $doc ) {
       $item = new SearchResultItem();
-      $item->id = $doc->id;
+      $item->id = $doc->id[0];
```

Comment:
Is this because the solr schema allows multiple id's? (same for titles and publication dates?)